astropy/units/tests/test_quantity.py

```diff
@@ -1349,3 +1349,16 @@ class MyQuantity(u.Quantity):
     assert type(q1) is u.Quantity
     q2 = u.Quantity(1., my_unit, subok=True)
     assert type(q2) is  MyQuantity
+
+def test_inserting_implicit_Quantity_into_Table():
+    from ... table import Table
```

Comment:
ps. on the location: this is the right file to insert the test, but maybe add it just below the `test_quantity_from_table` tests, since it is related to that. (indeed, you could also add something to that test).